modules/analysis/rosbag-image-pipeline/image_dags/ros_image_pipeline.py

```diff
@@ -414,15 +441,45 @@ def sagemaker_lanedet_operation(**kwargs):
         logger.info("All object detection jobs complete")
 
 
+def emr_batch_operation(**kwargs):
+    ds = kwargs["ds"]
+    batch_id = kwargs["dag_run"].run_id
+
+    JOB_DRIVER = {
+        "sparkSubmit": {
+            "entryPoint": f"{S3_SCRIPT_DIR}detect_scenes.py",
+            "entryPointArguments": [
+                "--batch-metadata-table-name", DYNAMODB_TABLE,
+                "--batch-id", batch_id,
+                "--bucket", TARGET_BUCKET,
+                "--region", REGION,
+                "--output-dynamo-table", SCENE_TABLE
+            ],
+            "sparkSubmitParameters": "--jars s3://addf-ros-image-demo-artifacts-bucket-75fe6115/dags/ros-image-demo/dags-aws/spark_scripts/spark-dynamodb_2.12-1.1.1.jar"
+        }
+    }
+
+    start_job_run_op = EmrServerlessStartJobOperator(
```

Comment:
This DAG has existed prior to EMR Serverless support...?  If so, is this change backward-compatible?  Will it break the existing pipeline usage?